python/gtsam/tests/test_Cal3Unified.py

```diff
@@ -139,6 +139,17 @@ def test_jacobian(self):
         self.gtsamAssertEquals(z, np.zeros(2))
         self.gtsamAssertEquals(H @ H.T, 4*np.eye(2))
 
+        Dcal = np.zeros((2, 10), order='F')
```

Comment:
Now that we *have* derivatives, we no longer have to jump through the linearize hoop above for the earlier test. Can you read and update the test?

Reply:
I think they test for different derivatives though